docs/04-onboarding/first-day.md

```diff
@@ -0,0 +1,67 @@
+# Your First Day at Bixal!
+
+Welcome to Bixal! We know you probably have lots of questions and we hope
+this guide will help you get answers, or at least where to look! First a few
+notes about this technology handbook!
+
+## Welcome to the Bixal Techbook
+
+This guide aims to provide you with a searchable first stop to find out anything
+you might need involving technology. This documentation is meant to be simple
+and easy to understand. It's primary purpose is for the average employee, not
+just those in the technology silo. If you are reading something in here and it
+seems out of place, confusing, or needs updating please file an issue at
+[https://github.com/bixal/techbook/issues](https://github.com/bixal/techbook/issues).
+If you are feeling particularly brave we gladly
+accept changes and pull requests! Ask any of the technology staff
+to show you how! You can do it from firefox or chrome, and don't
+need a lot of technology experince!
+
+## The Overall Process and What to Expect Your First Day
+
+### Things Every Employee Needs
+
+* Microsoft Office 365 Account
+  * Microsoft Teams is our primary chat and teleconfernce service.
```

Comment:
Should we mention go to meeting?

Reply:
no, I'm phasing it out for most things. We're just about to get the all hands on it as well. If they need an account we can get it for them through helpdesk